tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher.py

```diff
@@ -326,31 +326,59 @@ def _prepare_bitstream_firmware(self, config):
 
     @staticmethod
     def get_cpu_extension(cpu_extensions, selection_mode):
+        def get_cpu_extensions_list(file_format, base_name, selection_mode='',):
+            if not selection_mode:
+                default_cpu_extension = file_format.format(base_name)
+                extension_list = list(extensions_path.glob(default_cpu_extension))
+
+                if extension_list:
+                    return extension_list
+
+                cpu_info_flags = get_cpu_info()['flags']
+                supported_flags = ['avx512', 'avx2', 'sse4']
+                for flag in supported_flags:
+                    if flag in cpu_info_flags:
```

Comment:
Hold on. `py-cpuinfo` doesn't actually support a flag named `sse4`. It has `sse4_1` and `sse4_2`. So this change is going to break extension library selection on CPUs that don't support AVX2.

There could probably be a better fix for this, but for now, let's just revert to the original logic here. I don't want any regressions.

Reply:
I add a mapping from cpu_info flags to library suffixes, however, I do not know, has it ever worked on platforms without sse4_1 & sse4_2

Reply:
and also return previously used logic too (for sse4 as default suffix)